SonarQube.Common/AnalysisConfig/AnalysisSetting.cs

```diff
@@ -48,6 +48,20 @@ public class AnalysisSetting
         /// </remarks>
         private static readonly Regex ValidSettingKeyRegEx = new Regex(@"^\w[\w\d\.-]*$", RegexOptions.Compiled);
 
+        //TODO: this expression only works for single-line values
```

Comment:
Let's keep it like this for now and come back to this decision when need arises.